catalog/app/containers/Bucket/Overview.js

```diff
@@ -804,6 +812,11 @@ function Head({ req, s3, overviewUrl, bucket, description }) {
             fallback={() => null}
           />
         </M.Box>
+        <RRLink className={classes.settings} to={urls.adminBuckets(bucket)}>
```

Comment:
we should only show this for users with admin privileges

Reply:
done